chain/client/src/sync.rs

```diff
@@ -522,7 +522,7 @@ impl StateSync {
                         let prev = shard_sync_download.downloads[0].prev_update_time;
                         let error = shard_sync_download.downloads[0].error;
                         if now - prev > Duration::seconds(STATE_SYNC_TIMEOUT) || error {
-                            download_timeout = true;
+                            download_timeout = now - prev > Duration::seconds(STATE_SYNC_TIMEOUT);
```

Comment:
Why duplicate `now - prev > Duration::seconds(STATE_SYNC_TIMEOUT)`?

Reply:
why not? if A or B then if A { A happened }